src/utils.ts

```diff
@@ -15,10 +15,10 @@ export interface CaptionItem extends Cuepoint {
 // TODO: consider move to contrib
 export const parseExpandMode = (value: string): KitchenSinkExpandModes => {
```

Comment:
I see the method accepts string and not enum values so is it on purpose?
And in general if there are only two options and we translate enum to enum values do we even need this method?

Reply:
addressed